core/CMakeLists.txt

```diff
@@ -745,12 +745,12 @@ message(" ")
 message("Configuration on ${NOW} : ")
 message(" ")
 
-message("   Host:                         ${host} -- ${DISTNAME} ${DISTVER} ")
+message("   Host:                         ${host} -- ${PLATFORM} ${DISTVER} ")
 message(
   "   Bareos version:               ${BAREOS} ${BAREOS_FULL_VERSION} (${DATE}) "
 )
 message(
-  "   Distribution:                 ${DISTNAME}"
+  "   Build platform:               ${PLATFORM}"
```

Comment:
"Build Host" should also be common.

Reply:
I thought this through. For me a "Build Host" is one specific system (i.e. "my laptop"), thus I ended up with "Build platform" which will end up containing "redhat" or "debian" or "solaris".